src/pages/index.js

```diff
@@ -42,87 +79,188 @@ const popupImage = new PopupWithImage(
 popupImage.setEventListeners();
 
 // Functions
-
+// Render
 function renderCard(cardData, cardListEl) {
-  const card = new Card(cardData, "#card-template", handlePreviewImage);
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handlePreviewImage,
+    handleDeleteClick,
+    handleLikeClick,
+    userId
+  );
   return card.getCard();
 }
 
 function handlePreviewImage({ name, link }) {
   popupImage.open({ name, link });
 }
 
-// Event handlers
-
-const editPopup = new PopupWithForm("#profile-edit-modal", (inputValues) => {
-  userInfo.setUserInfo(inputValues);
-  // profileTitle.textContent = profileTitleName.value;
-
-  editPopup.close();
-});
-
-function handleAddCardFormSubmit(inputValues) {
-  const name = inputValues.title;
-  const link = inputValues.url;
+function handleProfileSubmit(data) {
+  editPopup.renderLoading(data);
+  api
+    .editProfileInformation(data)
+    .then(({ name, about }) => {
+      userInfo.setUserInfo(name, about);
+    })
+    .then(() => {
```

Comment:
Needs correction: You can do everything in one `then` block. Fix that here and for other requests